*** Starting uWSGI 2.0.19.1 (64bit) on [Fri Oct 16 17:03:43 2020] ***
compiled with version: 5.4.0 20160609 on 16 October 2020 08:55:48
os: Linux-4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/python/Desktop/projects/project/meiduo_mall
writing pidfile to uwsgi.pid
detected binary path: /home/python/.virtualenvs/meiduo/bin/uwsgi
chdir() to /home/python/Desktop/projects/project
your processes number limit is 11807
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:33762 (port auto-assigned) fd 3
Python version: 3.5.2 (default, Jul 17 2020, 14:04:10)  [GCC 5.4.0 20160609]
Set PythonHome to /home/python/.virtualenvs/meiduo
Python main interpreter initialized at 0x1faa9c0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
Traceback (most recent call last):
  File "meiduo_mall/meiduo_mall/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/site-packages/django/__init__.py", line 22, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/site-packages/django/conf/__init__.py", line 56, in __getattr__
    self._setup(name)
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/site-packages/django/conf/__init__.py", line 41, in _setup
    self._wrapped = Settings(settings_module)
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/site-packages/django/conf/__init__.py", line 110, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'meiduo_mall.settings'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8080)
spawned uWSGI worker 1 (pid: 8082, cores: 2)
spawned uWSGI worker 2 (pid: 8083, cores: 2)
spawned uWSGI worker 3 (pid: 8084, cores: 2)
spawned uWSGI worker 4 (pid: 8085, cores: 2)
spawned uWSGI http 1 (pid: 8086)
--- no python application found, check your startup logs for errors ---
[pid: 8082|app: -1|req: -1/1] 127.0.0.1 () {44 vars in 828 bytes} [Fri Oct 16 17:04:34 2020] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 8082|app: -1|req: -1/2] 127.0.0.1 () {42 vars in 755 bytes} [Fri Oct 16 17:04:34 2020] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 8086)
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.19.1 (64bit) on [Fri Oct 16 17:09:29 2020] ***
compiled with version: 5.4.0 20160609 on 16 October 2020 08:55:48
os: Linux-4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/python/Desktop/projects/project/meiduo_mall
writing pidfile to uwsgi.pid
detected binary path: /home/python/.virtualenvs/meiduo/bin/uwsgi
chdir() to /home/python/Desktop/projects/project
your processes number limit is 11807
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:36601 (port auto-assigned) fd 3
Python version: 3.5.2 (default, Jul 17 2020, 14:04:10)  [GCC 5.4.0 20160609]
Set PythonHome to /home/python/.virtualenvs/meiduo
Python main interpreter initialized at 0x1b1f9c0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
Traceback (most recent call last):
  File "meiduo_mall/meiduo_mall/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/site-packages/django/__init__.py", line 22, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/site-packages/django/conf/__init__.py", line 56, in __getattr__
    self._setup(name)
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/site-packages/django/conf/__init__.py", line 41, in _setup
    self._wrapped = Settings(settings_module)
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/site-packages/django/conf/__init__.py", line 110, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/home/python/.virtualenvs/meiduo/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'meiduo_mall.settings'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8521)
spawned uWSGI worker 1 (pid: 8523, cores: 2)
spawned uWSGI worker 2 (pid: 8524, cores: 2)
spawned uWSGI worker 3 (pid: 8525, cores: 2)
spawned uWSGI worker 4 (pid: 8526, cores: 2)
spawned uWSGI http 1 (pid: 8527)
--- no python application found, check your startup logs for errors ---
[pid: 8523|app: -1|req: -1/1] 127.0.0.1 () {44 vars in 827 bytes} [Fri Oct 16 17:09:40 2020] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 8523|app: -1|req: -1/2] 127.0.0.1 () {42 vars in 754 bytes} [Fri Oct 16 17:09:40 2020] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 8527)
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
